pkg/kubelet/kubelet.go

```diff
@@ -705,31 +704,6 @@ func containerRef(pod *api.Pod, container *api.Container) (*api.ObjectReference,
 	return ref, nil
 }
 
-// setRef stores a reference to a pod's container, associating it with the given docker id.
```

Comment:
can we move containerRef() to the manager too? Something like: GenerateContainerRef(pod, container). We should re-examine how we use this whole structure. The weird create and cache behavior of today may be simplified (but for future work :) )